crates/profiler/src/trace_reader.rs

```diff
@@ -126,27 +126,17 @@ pub fn collect_samples_from_trace(trace: &CallTrace) -> Vec<Sample> {
 }
 
 pub struct ResourcesKeys {
-    pub builtins: HashSet<String>,
-    pub syscalls: HashSet<DeprecatedSyscallSelector>,
+    pub keys: HashSet<String>,
 }
 
 impl ResourcesKeys {
     pub fn measurement_types(&self, context: &mut ProfilerContext) -> Vec<ValueType> {
```

Comment:
I still have no idea what this function returns.

Reply:
https://github.com/google/pprof/blob/main/proto/README.md#measurement-values

Didn't want to name it measurement values since those are types imo. Also the return type tells you the story

Reply:
It doesn't tell the story, i have no idea what this function does looking at the name.